taxonomy-de.json

```diff
@@ -0,0 +1,443 @@
+      "example":"Jane Rotonda for \"The Larry Meiller Show"
+   },
+   {
+      "key":"Creative Direction\/Senior Producer",
+      "group":"Kreative Leitung",
+      "role":"Senior Produzent",
+      "description":"Der Senior Produzent ist der zweitälteste Produzent der Produktion (an zweiter Stelle nach dem Executive Producer). Er beaufsichtigt die Produzenten und die allgemeine Leitung und Logistik der gesamten Produktion.",
+      "example":"Dr. Jeremy Weisz from \"INspired INsider\""
+   },
+   {
+      "key":"Creative Direction\/Producer",
+      "group":"Kreative Leitung",
+      "role":"Produzent",
+      "description":"Der Produzent koordiniert und führt die Produktion des Podcasts durch. Zu seinen Aufgaben können die Unterstützung bei der Erstellung der kreativen Leitung eines Projekts, die Budgetierung, Recherche, Terminierung und die Überwachung der Bearbeitung und Endproduktion gehören.",
+      "example":""
+   },
+   {
+      "key":"Creative Direction\/Associate Producer",
+      "group":"Kreative Leitung",
+      "role":"Associate Producer",
+      "description":"Der Associate Producer führt eine oder mehrere Producer-Funktionen aus, die ihm von einem Produzenten übertragen wurden.",
+      "example":"Alex Baumhardt for \"APM Reports\""
+   },
+   {
+      "key":"Creative Direction\/Development Producer",
+      "group":"Kreative Leitung",
+      "role":"Development Producer",
+      "description":"Der Development Producer koordiniert und führt die Vorproduktionsleitung eines Podcasts aus. Zu seinen Aufgaben gehört es, neue Episoden- und Serienideen zu finden und mit Autoren und Forschern zusammenzuarbeiten, um das Konzept für die Produktion vorzubereiten.",
+      "example":""
+   },
+   {
+      "key":"Creative Direction\/Creative Director",
+      "group":"Kreative Leitung",
+      "role":"Creative Director",
+      "description":"Der Creative Director ist für die kreative Strategie und Ausführung einer ganzen Serie verantwortlich. Oft erstreckt sich diese Rolle über den Inhalt hinaus und betrifft begleitende Grafiken, Musik, Marketingkampagnen und mehr.",
+      "example":"Neil Druckmann on \"The Official The Last of Us\""
+   },
+   {
+      "key":"Cast\/Host",
+      "group":" Besetzung",
+      "role":"Moderator",
+      "description":"Der Moderator ist der On-Air-Zeremonienmeister des Podcasts und ständig in jeder Folge präsent (mit Ausnahme von Gastmoderatoren und alternativen Folgen). Die Aufgaben des Moderators können das Führen von Interviews, das Vorstellen von Geschichten und Segmenten, Erzählen und mehr. Es kann mehr als einen Host pro Podcast oder Folge geben.",
+      "example":"Joe Rogan for \"The Joe Rogan Experience\""
+   },
+   {
+      "key":"Cast\/Co-Host",
+      "group":" Besetzung",
```

Comment:
@CTHTC Why is there a space character before Besetzung and Schreiben?

Reply:
Its a mistake. The space must be deleted.

Reply:
I have sent a new file version.